draft-ietf-quic-recovery.md

```diff
@@ -615,6 +615,8 @@ are described in this section.
 bytes_in_flight:
 : The sum of the size in bytes of all sent packets that contain at least
   one retransmittable frame, and have not been acked or declared lost.
+  The size does not include IP or UDP overhead.  Ack only frames do not
+  count towards byte_in_flight.
```

Comment:
Isn't this basically the same question as the question which packets should be congestion controlled? All bytes that are congestion controlled should be counted for bytes_in_flight, no?